xb_scorm/xb_scorm.py

```diff
@@ -65,13 +65,17 @@ def max_score(self):
     tc_statements = Dict(default={}, scope=Scope.user_state)  # statementId -> JSON document
 
     def __init__(self, *args, **kwargs):
-        XBlock.__init__(self, *args, **kwargs)
+        try:
```

Comment:
Removed __init__ method,

http://edx-code.narkive.com/6bsrps1W/xblock-overriding-the-init-method